tests/test_lsl_stream.py

```diff
@@ -21,91 +22,95 @@
 ) = nm_IO.read_BIDS_data(
     PATH_RUN=PATH_RUN, BIDS_PATH=PATH_BIDS, datatype=datatype
 )
-f_name = f"{PATH_RUN}_ieeg.vhdr"
     
-def test_general_lsl():
-    if __name__ == "__main__":
+def test_mne_lsl():
```

Comment:
Please provide a docstring comment what this test does

Reply:
Alright I will add docstring comments for the tests 👍🏽